crates/telio-traversal/src/endpoint_providers/upnp/mod.rs

```diff
@@ -18,52 +21,83 @@ use telio_sockets::External;
 use telio_task::{io::chan::Tx, task_exec, BoxAction, Runtime, Task};
 use telio_utils::{
     exponential_backoff::{Backoff, ExponentialBackoff, ExponentialBackoffBounds},
-    telio_log_debug, telio_log_info, PinnedSleep,
+    telio_log_debug, telio_log_info, telio_log_warn, PinnedSleep,
 };
 use telio_wg::{DynamicWg, WireGuard};
-use tokio::{net::UdpSocket, sync::Mutex};
+use tokio::{net::UdpSocket, sync::Mutex, time::sleep};
 
 #[cfg(test)]
 use mockall::automock;
 #[cfg(test)]
-const UPNP_INTERVAL: Duration = Duration::from_millis(100);
+const UPNP_INTERVAL_MS: u64 = 100;
 
 const MAX_SUPPORTED_PACKET_SIZE: usize = 1500;
 #[cfg(not(test))]
-const UPNP_INTERVAL: Duration = Duration::from_secs(5);
-const GET_INTERFACE_TIMEOUT: u64 = 2;
+const UPNP_INTERVAL_MS: u64 = 5000;
```

Comment:
Same for this one

Reply:
To match the type defined by the other one.